debian/control

```diff
@@ -1,16 +1,16 @@
 Source: switchboard-plug-power
 Section: x11
 Priority: optional
-Maintainer: Cody Garver <cody@elementaryos.org>
+Maintainer: elementary OS Team <builds@elementaryos.org>
```

Comment:
Use elementary.io email instead of elementaryos.org. Assuming either of these email addresses work.